src/employees/entities/employee.entity.ts

```diff
@@ -7,53 +7,53 @@ export type EmployeeDocument = Employee & Document;
 export class Employee {
   @Prop({ required: true, unique: true })
   employeeNo: string;
-  @Prop()
+  @Prop({ required: true })
   firstName: string;
-  @Prop()
+  @Prop({ required: true })
   lastName: string;
   @Prop()
   middleName: string;
-  @Prop()
-  fsCode: string;
-  @Prop()
-  bioCode: string;
-  @Prop()
+  @Prop({ required: true })
   position: string;
-  @Prop()
+  @Prop({ required: true })
   rank: string;
-  @Prop()
-  division: string;
-  @Prop()
+  @Prop({ required: true })
   department: string;
-  @Prop()
-  designation: string;
-  @Prop()
+  @Prop({ required: true })
+  location: string;
+  @Prop({ required: true })
+  isActive: boolean;
+  @Prop({ required: true })
+  userGroup: string;
+  @Prop()
+  reportsTo: string;
+  @Prop({ required: true })
   dateHired: Date;
-  @Prop()
-  yearsInService: number;
-  @Prop()
+  @Prop({ required: true })
   employmentStatus: string;
   @Prop()
   endOfProbationary: Date;
   @Prop()
   contractEndDate: Date;
-  @Prop()
+  @Prop({ required: true })
   gender: string;
-  @Prop()
+  @Prop({ required: true })
   birthDate: Date;
-  @Prop()
-  age: number;
-  @Prop()
+  @Prop({ required: true })
   contactNumber: string;
-  @Prop()
+  @Prop({ required: true })
   taxExemption: string;
   @Prop({ required: true, unique: true })
-  email: string;
-  @Prop()
+  companyEmail: string;
+  @Prop({ required: true, unique: true })
+  personalEmail: string;
+  @Prop({ required: true })
   backAccountNo: string;
```

Comment:
Change to payrollBankAccount, type is object
Sample: {
"accountName": "",
"accountNumber": "",
"bankName": "",
"bankBranch": "",
}